backends/xnnpack/operators/TARGETS

```diff
@@ -13,6 +13,6 @@ runtime.python_library(
         "//executorch/backends/xnnpack/utils:xnnpack_utils",
         "//executorch/exir:graph_module",
         "//executorch/exir/backend:backend_details",
-        "//executorch/extension/aot_util:aot_util",
+        "//executorch/util/aot:aot_util",
```

Comment:
We're trying to kill the `util` directory; please keep this under `extension`.

Reply:
I'll move it. I feel like it might not belong under extension, just because it's not really an extension, it's ideally part of the core runtime code (tied to the logic under exir). There's not really a great place for aot native code, though I'll just put it back under extension for now.